data/human/cim-outfits.txt

```diff
@@ -0,0 +1,29 @@
+outfit "Blaze-Pike Turret (modified)"
```

Comment:
all human outfits are in outfits.txt, and for one outfit it's really not worth spliting (ik, was a plugin, no worries)